lib/utils/createDomain.js

```diff
@@ -9,8 +9,12 @@ function createDomain(options, server) {
   // ipv6 detection is not required as 0.0.0.0 is just used as a placeholder
   let hostname;
 
-  if (options.useLocalIp) {
-    hostname = ip.v4.sync() || '0.0.0.0';
+  if (options.host === 'local-ip') {
+    hostname = ip.v4.sync() || ip.v6.sync();
+  } else if (options.host === 'local-ipv4') {
+    hostname = ip.v4.sync();
+  } else if (options.host === 'local-ipv6') {
+    hostname = ip.v6.sync();
```

Comment:
What about of `ipv6` is not available, what is the error? Maybe we need improve error message, need test, at least manually

Reply:
Same for ipv4

Reply:
It's like this, for now, WIP on improvements.

<img width="742" alt="Screenshot 2021-03-26 at 7 44 31 PM" src="https://user-images.githubusercontent.com/46647141/112644761-c56f9080-8e6b-11eb-90dd-67b5f544b886.png">